src/clients/workaround.ts

```diff
@@ -4,7 +4,9 @@ import Asm from 'stream-json/Assembler.js';
 import streamJsonParser from 'stream-json/Parser.js';
 import { ReadableStream } from 'stream/web';
 
-import { PostCommandUpdateFileData } from '@figpot/src/clients/penpot';
+
+
```

Comment:
Lines to remove, did you make vscode using extensions listing in `extensions.json`? With this, prettier should use local project settings for formatting. If not, use `npm run format`